management/commands/aggregate_subject_trials.py

```diff
@@ -280,7 +378,12 @@ def handle(self, *_, **options):
 
         if not rerun:
             logger.info('Aggregate hash unchanged; exiting')
-            return None, None, None
+            if compute_training_status:
+                # The trials table may not need to be rerun, but we need to check if the training status table exists
+                training_dset, training_out_file = self.handle_training_status(trials_table=None, rerun=rerun, dry=dry)
+                return training_dset, training_out_file, None
```

Comment:
The output seems to vary a lot. Can we keep at consistent with` (trials_dset, training_dset), (trials_file, training_file), log_file`?